brightStars_DM_sprint_Feb2020/BeyondRadius.ipynb

```diff
@@ -0,0 +1,422 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for tract, dens in zip(tractList, densities):\n",
+    "    # average detection densities around stars in this magnitude bin:\n",
+    "    magDen = np.mean(dens,axis=0)\n",
+    "    magDenStd = np.std(dens,axis=0)\n",
+    "    plt.errorbar(distanceToEdge, magDen, yerr=magDenStd, capsize=2)\n",
+    "    plt.plot(distanceToEdge, np.min(dens,axis=0), c='red', ls='--')\n",
+    "    plt.plot(distanceToEdge, np.max(dens,axis=0), c='red', ls='--')\n",
+    "\n",
+    "    plt.title(\"Tract {}\\n\".format(tract)+\" ({} bright stars)\".format(len(dens)))\n",
+    "    plt.xlabel('Distance to ABOUT 1/6th of mask edge (arcsec)')\n",
+    "    plt.ylabel(r'Detection density (arcsec$^{-2}$)')\n",
+    "    \n",
+    "    if savePlots:\n",
+    "        plt.savefig(\"plots/\" + plotStub + \"tract{}\".format(tract) + plotExt)\n",
+    "    plt.show()\n",
+    "    plt.close()"
+   ]
+  }
+ ],
+ "metadata": {
```

Comment:
I'm not sure of the most appropriate place to put this, but what do you conclude from this analysis?  Your slides didn't provide any assessment either...